azure-sdk/src/test/groovy/io/micronaut/azure/condition/EnvironmentCredentialsConditionSpec.groovy

```diff
@@ -12,7 +12,7 @@ import static com.azure.core.util.Configuration.PROPERTY_AZURE_TENANT_ID
 import static com.azure.core.util.Configuration.PROPERTY_AZURE_USERNAME
 import static com.github.stefanbirkner.systemlambda.SystemLambda.withEnvironmentVariable
 
-@IgnoreIf({ jvm.java17 })
+@IgnoreIf({ sys['java.version'] >= '1.9' })
```

Comment:
does it mean we ignore this test always?

Reply:
I think it does... 😢 

And it's doing a string comparison of version numbers 🤔 

I think we need to remove this `@IgnoreIf` and add

```
tasks.withType(Test::class) {
    // required for SystemLambda
    jvmArgs = listOf("--add-opens", "java.base/java.util=ALL-UNNAMED")
}
```

to `azure-sdk/build.gradle.kts`